react/components/ProductSummaryImage/ProductImage.js

```diff
@@ -137,6 +147,8 @@ const ProductImageContent = ({
   displayMode,
   mainImageLabel,
   hoverImageLabel,
+  hoverImageIndex,
```

Comment:
Sounds good

Reply:
So it would be something like:

`hoverImage: { criteria: 'index', index: 3 }` or `hoverImage: { criteria: 'label', label: 'foobar' }`

The contentSchema could be something like:

https://rjsf-team.github.io/react-jsonschema-form/#eyJmb3JtRGF0YSI6eyJob3ZlckltYWdlIjp7ImNyaXRlcmlhIjoiaW5kZXgifSwiZmlyc3ROYW1lIjoiQ2h1YyIsImxhc3ROYW1lIjoiTm9ycmlzIiwiYWdlIjo3NSwiYmlvIjoiUm91bmRob3VzZSBraWNraW5nIGFzc2VzIHNpbmNlIDE5NDAiLCJwYXNzd29yZCI6Im5vbmVlZCJ9LCJzY2hlbWEiOnsicHJvcGVydGllcyI6eyJob3ZlckltYWdlIjp7InR5cGUiOiJvYmplY3QiLCJwcm9wZXJ0aWVzIjp7ImNyaXRlcmlhIjp7ImVudW0iOlsiaW5kZXgiLCJsYWJlbCJdfX0sImRlcGVuZGVuY2llcyI6eyJjcml0ZXJpYSI6eyJvbmVPZiI6W3sicHJvcGVydGllcyI6eyJjcml0ZXJpYSI6eyJlbnVtIjpbImluZGV4Il19LCJpbmRleCI6eyJ0eXBlIjoibnVtYmVyIn19fSx7InByb3BlcnRpZXMiOnsiY3JpdGVyaWEiOnsiZW51bSI6WyJsYWJlbCJdfSwibGFiZWwiOnsidHlwZSI6InN0cmluZyJ9fX1dfX19fX0sInVpU2NoZW1hIjp7ImZpcnN0TmFtZSI6eyJ1aTphdXRvZm9jdXMiOnRydWUsInVpOmVtcHR5VmFsdWUiOiIiLCJ1aTphdXRvY29tcGxldGUiOiJmYW1pbHktbmFtZSJ9LCJsYXN0TmFtZSI6eyJ1aTplbXB0eVZhbHVlIjoiIiwidWk6YXV0b2NvbXBsZXRlIjoiZ2l2ZW4tbmFtZSJ9LCJhZ2UiOnsidWk6d2lkZ2V0IjoidXBkb3duIiwidWk6dGl0bGUiOiJBZ2Ugb2YgcGVyc29uIiwidWk6ZGVzY3JpcHRpb24iOiIoZWFydGhpYW4geWVhcikifSwiYmlvIjp7InVpOndpZGdldCI6InRleHRhcmVhIn0sInBhc3N3b3JkIjp7InVpOndpZGdldCI6InBhc3N3b3JkIiwidWk6aGVscCI6IkhpbnQ6IE1ha2UgaXQgc3Ryb25nISJ9LCJkYXRlIjp7InVpOndpZGdldCI6ImFsdC1kYXRldGltZSJ9LCJ0ZWxlcGhvbmUiOnsidWk6b3B0aW9ucyI6eyJpbnB1dFR5cGUiOiJ0ZWwifX19LCJ0aGVtZSI6ImRlZmF1bHQiLCJsaXZlU2V0dGluZ3MiOnsidmFsaWRhdGUiOmZhbHNlLCJkaXNhYmxlIjpmYWxzZSwib21pdEV4dHJhRGF0YSI6ZmFsc2UsImxpdmVPbWl0IjpmYWxzZX19